CONTRIBUTING.md

```diff
@@ -27,12 +27,9 @@ OCI discussion happens on #opencontainers on [Freenode][] ([logs][irc-logs]).
 
 ## Git
 
-### Security issues
+### Security
 
-If you are reporting a security issue, do not create an issue or file a pull
-request on GitHub. Instead, disclose the issue responsibly by sending an email
-to security@opencontainers.org (which is inhabited only by the maintainers of
-the various OCI projects).
+If you wish to report a security issue, please see the protocol under [security][security].
```

Comment:
nit...
/s/see the protocol under [security]/follow the [OCI org security issue reporting procedure]/